src/plugins/melpa-resolver.js

```diff
@@ -0,0 +1,82 @@
+const Joi = require('joi');
+const findReachableUrls = require('find-reachable-urls');
+const got = require('got');
+const insight = require('../utils/insight');
+
+let lastModified = 'Mon, 19 Jun 2017 15:27:49 GMT';
```

Comment:
I wonder if there is any downside to hardcoding this date. Probably the right thing to do, but it makes my radar go off seeing it like this.